sormas-api/src/main/java/de/symeda/sormas/api/sample/PathogenTestDto.java

```diff
@@ -57,31 +57,31 @@
 	public static final String REPORT_DATE = "reportDate";
 	public static final String VIA_LIMS = "viaLims";
 
-	@Required
+	@NotNull
 	private SampleReferenceDto sample;
-	@Required
+	@NotNull
 	private Disease testedDisease;
 	private String testedDiseaseDetails;
 	private String typingId;
-	@Required
+	@NotNull
 	private PathogenTestType testType;
 	@SensitiveData
 	private String testTypeText;
-	@Required
+	@NotNull
 	private Date testDateTime;
-	@Required
+	@NotNull
 	private FacilityReferenceDto lab;
 	@SensitiveData
 	private String labDetails;
-	@Required
+	@NotNull
 	@SensitiveData
 	private UserReferenceDto labUser;
-	@Required
+	@NotNull
 	private PathogenTestResultType testResult;
-	@Required
+	@NotNull
```

Comment:
`testResultText` seems to be marked here as NotNull, however on the mobile side it's not marked or enforced to be required, which results in a `400` when synchronizing Casese or Samples

Reply:
fixed